src/main/java/uk/gov/hmcts/darts/common/repository/EventRepository.java

```diff
@@ -53,6 +53,7 @@ SELECT EXISTS (
          SELECT new uk.gov.hmcts.darts.event.model.EventSearchResult(
             e.id,
             e.createdDateTime,
```

Comment:
Would there be another PR to remove createdDateTime, ChronicleID and Antecedent ID from this payload?
You've removed it from the payload for the event endpoint further down, so wondered why that's been included in this PR and not the future one?